src/transactions/financial.ts

```diff
@@ -245,7 +290,7 @@ const buildTransferCreatorCoinMetadata = (
   return metadata;
 };
 
-export const constructTransferCreatorCoinTransaction = (
+const constructTransferCreatorCoinTransaction = (
```

Comment:
Any reason we don't want to export this anymore?

Reply:
it wasn't getting used anywhere outside if here.